defaults/04_redhat_marketplace.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: "operators.coreos.com/v1"
+kind: "OperatorSource"
+metadata:
+  name: "redhat-marketplace"
+  namespace: "openshift-marketplace"
+  labels:
+    opsrc-provider: redhat
```

Comment:
I think that this needs to be `marketplace` or something similar so that we can tell it is different than the default RedHat catalog